src/Context/LoginContext.php

```diff
@@ -53,6 +54,26 @@ public function iAmLoggedInWithPermissions($permCode)
         $this->stepILogInWith($email, $password);
     }
 
+    /**
+     * Creates and login as a member of a group with the correct permissions.
+     * Example: Given I am logged in as a memeber of "ADMIN" group
+     *
+     * @Given /^I am logged in as a memeber of "([^"]*)" group$/
+     * @param string $permCode
+     */
+    public function iAmLoggedInAsMemeberOfGroup($permCode)
+    {
+        $group = Group::get()->filter('Title', "$permCode group")->first();
```

Comment:
```suggestion
    public function iAmLoggedInAsMemeberOfGroup($groupName)
    {
        $group = Group::get()->filter('Title', "$groupName")->first();
```
Remove the necessity to have "group" at the end of the group name.